src/styles/blocks/card.scss

```diff
@@ -0,0 +1,74 @@
+.card {
+  width: $CardWidth;
```

Comment:
The variable `$CardWidth` should be defined in your variables file or imported correctly. Ensure that it is declared and accessible here.